config/config.go

```diff
@@ -59,6 +60,7 @@ func defaultConfig() {
 	Global.Parsers.Loaded = "spack,shpc"
 	Global.Branch.Prefix = "binoc/"
 	Global.PR.IgnoreLabel = "binoc-blacklist"
+	Global.PR.Skip = "false"
```

Comment:
Let's actually make the default for people not running Binoc as an action `Global.PR.Skip = true`. So that Binoc can be run as an independent workflow from GitHub.